tests/vmi_headless_test.go

```diff
@@ -115,7 +116,9 @@ var _ = Describe("[rfe_id:609]VMIheadless", func() {
 				memDiff := normalComputeContainer.Resources.Requests.Memory()
 				memDiff.Sub(*computeContainer.Resources.Requests.Memory())
 
-				Expect(memDiff.ScaledValue(resource.Mega) > 15).To(BeTrue(), "memory difference between headless and normal should be roughly 16M")
+				Expect(memDiff.ScaledValue(resource.Mega) > 15).To(BeTrue(),
+					fmt.Sprintf("memory difference between headless (%s) and normal (%s) should be roughly 16M",
+						computeContainer.Resources.Requests.Memory(), normalComputeContainer.Resources.Requests.Memory()))
```

Comment:
nit: Why not go all the way and print the actual difference as well ?

Reply:
To me, the diff is not helpful for debug. I need to understand which of the two is too big or too little.

Reply:
@ezrasilvera did I convince you? If not, can you suggest a text that would satisfy you?

Reply:
@dankenigsberg I just wanted to be sure that indeed you don't think it's needed.  Your warning and test is about a **diff** value, while the printouts are of the individual values. So basically the tester/user will calculate it to see what happen (unless one of the values is really an outlier). 
I thought of something like:
`memory difference between headless (%s) and normal (%s), is %sM  but should be roughly 16M`
If you have strong opinion against it, leave it as is and replay, it's not a big deal ..

Reply:
I find this copious and annoying, but review in king.